src/main/java/flab/gotable/service/StoreService.java

```diff
@@ -0,0 +1,144 @@
+
+        // 예약 가능 날짜
+        Map<String, DayInfo> availableDays = new HashMap<>();
+        int daysAdded = 0;
+
+        for (int i = 0; daysAdded < store.getMaxAvailableDay(); i++) {
+            LocalDate targetDate = LocalDate.now().plusDays(i);
+
+            // 해당 날짜의 영업 스케줄 결정
+            final WorkSchedule workSchedule = getWorkSchedule(targetDate, specificSchedules, dailySchedules);
+            final String splitTime = getSplitTime(targetDate, specificSchedules, dailySchedules);
+
+            if (workSchedule != null && splitTime != null) {
+                daysAdded++;
+                // 선택 가능 시간 계산
+                List<String> selectableTimes = calculateSelectableTimes(workSchedule, splitTime);
+
+                // 사용 가능 날짜 추가
+                DayInfo dayInfo = new DayInfo(workSchedule, selectableTimes);
+                availableDays.put(targetDate.toString(), dayInfo);
+            }
+        }
+
+        String openSchedule = calculateOpenSchedule(dailySchedules);
+
+        return new StoreDetailsResponseDto(
+                store.getId(),
+                store.getName(),
+                store.getAddress(),
+                store.getMaxMemberCount(),
+                store.getMaxAvailableDay(),
+                openSchedule,
+                availableDays
+        );
+    }
+
+    private WorkSchedule getWorkSchedule(LocalDate targetDate, List<SpecificSchedule> specificSchedules, List<DailySchedule> dailySchedules) {
+        final SpecificSchedule specificSchedule = findSpecificSchedule(targetDate, specificSchedules);
+
+        if (specificSchedule != null) {
+            return new WorkSchedule(specificSchedule.getOpenTime(), specificSchedule.getCloseTime());
+        } else {
+            final DailySchedule dailySchedule =findDailySchedule(targetDate, dailySchedules);
+
+            if (dailySchedule != null) {
+                return new WorkSchedule(dailySchedule.getOpenTime(), dailySchedule.getCloseTime());
+            } else {
+                return null;
+            }
+        }
+    }
+
+    private String getSplitTime(LocalDate targetDate, List<SpecificSchedule> specificSchedules, List<DailySchedule> dailySchedules) {
+        final SpecificSchedule specificSchedule = findSpecificSchedule(targetDate, specificSchedules);
+
+        if (specificSchedule != null) {
+            return specificSchedule.getSplitTime();
+        } else {
+            final DailySchedule dailySchedule =findDailySchedule(targetDate, dailySchedules);
+
+            if (dailySchedule != null) {
+                return dailySchedule.getSplitTime();
+            } else {
+                return null;
+            }
+        }
+    }
+
+    private SpecificSchedule findSpecificSchedule(LocalDate targetDate, List<SpecificSchedule> specificSchedules) {
+        return specificSchedules.stream()
+                .filter(s -> s.getDate().equals(targetDate))
+                .findFirst()
+                .orElse(null);
+    }
```

Comment:
null 가능성이 있는 메서드/상태에 `@Nullable` 어노테이션을 붙여두면 코드편집기에서 해당 메서드가 null 가능성이 있다는 경고를 띄워주며, null처리를 하도록 도와줍니다.
NPE 방지를 위해 null을 리턴할 수 있는 메서드에는 위 어노테이션을 붙여주세요.
```suggestion
    @Nullable
    private SpecificSchedule findSpecificSchedule(LocalDate targetDate, List<SpecificSchedule> specificSchedules) {
        return specificSchedules.stream()
                .filter(s -> s.getDate().equals(targetDate))
                .findFirst()
                .orElse(null);
    }
```